easybuild/easyconfigs/h/HDF5/HDF5-1.8.19-foss-2017a.eb

```diff
@@ -0,0 +1,21 @@
+name = 'HDF5'
+version = '1.8.19'
+
+homepage = 'https://support.hdfgroup.org/HDF5/'
+description = """HDF5 is a unique technology suite that makes possible the management of
+ extremely large and complex data collections."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(version_major_minor)s/hdf5-%(version)s/src']
+
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['7f568e2464d4ab0a74d16b23956d900b']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Szip', '2.1.1'),
```

Comment:
@jhein32 Please also bump `Szip` to `2.1.1` in `HDF5-1.8.19-intel-2017a.eb` (merged via #5003), to keep those easyconfigs in sync.